app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  def update_posts_likes_counter
+    post.increment!(:likes_counter)
+    post.save
```

Comment:
- [ ] it will automatically save and update the database, you don't need to use the `save` method again.